list_test.py

```diff
@@ -0,0 +1,114 @@
+from list import List
+
+
+def create_list():
+    l = List()
+
+    l.push('p')
+    l.push('o')
+    l.push('n')
+    l.push('y')
+
+    return l
```

Comment:
The method `List.from_array` exists to simplify the syntax of list creation. This way it's possible to create a list with known 4 elements in one line of code ✔️ 

```suggestion
    return List.from_array(['p', 'o', 'n', 'y'])
```

Reply:
Replaced 4 `push` methods with one `from_array` method: https://github.com/nitkach/data_structures/blob/e111bf1e1a9f6aeba43e1a40a440d30304c69989/list_test.py#L5